pkg/keptn/resource_service.go

```diff
@@ -51,7 +52,47 @@ func (r V1ResourceHandler) GetResource(resource string, gitCommitID string) ([]b
 		queryParam = api.AppendQuery(url.Values{})
 	}
 
-	resourceContent, err := r.ResourceHandler.GetResource(*scope, queryParam)
+	return queryParam
+}
+
+// GetServiceResource returns the contents of a resource for a given gitCommitID
+func (r V1ResourceHandler) GetServiceResource(resource string, gitCommitID string) ([]byte, error) {
```

Comment:
Maybe we should document that an empty `gitCommitID` equals getting the resource without `gitCommitID`, or we could also make the parameter optional, e.g. (`...string`).

Reply:
I'd rather have it in here, same as keptn/go-utils does.
I would however document that when specifying an emtpy gitCommitID, Keptn will (most likely) just return the latest version of the file.

Reply:
Ok sure